### ISSUE TITLE
Adding Dioula language 

### ISSUE THREAD

**Issue by ismo168** [common-voice/common-voice#3663]
# Welcome to the Common Voice Community !

> Common Voice aims to make speech technology accessible to everyone by building an open sourced dataset of labelled voice data that is representative of languages, variants and accents spoken across the world. This template helps us to know how your language could participate in the Common Voice Project. There are three sections of this form, once you have filled out a section please click the checkbox. If you have any issues please contact <commonvoice@mozilla.org>.

## Pontoon Set-up

> To start a language on Common Voice volunteers localise our platform [via Pontoon](https://pontoon.mozilla.org/projects/common-voice/) and create sentence corpus’ of cc0 text. </p>

**Language name**

**Language code**

**Language size**

**Plural forms**

**Pontoon manager**

**Language Script**

## Sentence Collection Requirements

> On the Common Voice Platform contributors on the platform read out public domain sentences generated through sentence collection. Sentence collection is a crucial part in launching languages on Common Voice. To support the equitable participation of languages of Common Voice we have introduced [three new sentence collection requirements bands](https://discourse.mozilla.org/t/share-your-views-nuancing-sentence-collection-requirements-new-sentence-collection-bands/93134).

**Sentence Requirement Band**

- [ ] Band A

- [ ] Band B

- [ ] Band C

## Creating Community 

1. [optional to share] Why do you want to take part in Common Voice ?

2. [optional to share] Would you like to have a follow up conversation regarding community building ?

**Comment by ismo168** [comment]
Dear @ftyers 
Language name: Dioula
Language code: dyu
Language size: spoken by about 20 million people in Mali, Côte d'Ivoire and Burkina Faso (where Dioula has national language status).

Plural forms: yes it is similar to Bambara
[Pontoon](https://pontoon.mozilla.org/) manager: Yes I created an account but how to link it to my profile page here?
Language script: yes  N'Ko scripts is preferred

thank you

**Comment by ftyers** [comment]
Thanks for getting back to us so quickly!

- In terms of Plural forms, that means *other* (like [Bambara](https://pontoon.mozilla.org/bm/)). 
- In terms of linking your profile, 

![imatge](https://user-images.githubusercontent.com/449545/163599126-f136dcff-1a95-41de-af6f-9e4f1514f715.png)

First click on the little icon at the top right, then click on your email address. It should take you to a page that looks something like [this one](https://pontoon.mozilla.org/contributors/h4xjjwrfOv9TvtX0dZWfg2ED5SM/). Please paste that link here.

Let me know if I can help in any other way. There may be a delay as it is currently a holiday period and most of the team that deals with this is on holiday until the 19th April.

**Comment by ismo168** [comment]
Hello @ftyers @Heyhillary 
Thank you for your feedbacks , Is it the link ? 

https://github.com/common-voice/common-voice/issues/3663#issuecomment-1100231619

**Comment by ftyers** [comment]
Dear @ismo168, that is a link to the comment where I describe my profile. The link should start with https://pontoon.mozilla.org/contributors/.

**Comment by ismo168** [comment]
Dear @ftyers 
This ?  https://pontoon.mozilla.org/contributors/e9jGEUdc9nffc73RRfZJJpC6Ttc/

**Comment by ftyers** [comment]
Yes exactly, that's it. Now it's the weekend, and the person who manages accounts on Pontoon is away, but we'll get to it on Monday. Thanks again for your interest!

**Comment by ismo168** [comment]
Dear @ftyers @Heyhillary 
I read the guidelines. I have a few questions  
1. Is it possible to load existing audios and it corresponding text? collected from radios and TVs for example? 
2. I would like to know if there is a video explaining how to load content. 
Thank you

**Comment by ismo168** [comment]
@Heyhillary 
Thank you for your answer ! Yes I would like to talk about it if possible according to your availabilities

**Comment by Heyhillary** [comment]
I'm available Wednesday 15th June between 10am to 12pm UTC+1 could you email the Common Voice inbox which is commonvoice@mozilla.com so we can organise a time and I can send zoom details please ?

**Comment by Heyhillary** [comment]
Hey @ismo168 

How are you finding the project so far ! 

Do you need any support ?

I would encourage you to join our [community chat](https://chat.mozilla.org/#/room/#common-voice:mozilla.org)

**Comment by ismo168** [comment]
Hello @Heyhillary, 
Thank you for your question. I'm working together on the minimum required sentences for the language launch. I will get back to you very soon. I just joined the [community chat](https://chat.mozilla.org/#/room/#common-voice:mozilla.org)

**Comment by ismo168** [comment]
Dear @ftyers @Heyhillary
I've gathered sentences to be loaded. Since the language is not yet localized, I want to know how the user interface should be translated. I would also like to change N'ko to Latin script.
Thank you for your support.
